### PR TITLE
Ui overhaul

### PR DESCRIPTION
**Pull request by Joeper214**
## Goal

- UI refactor on every index pages
- Avoid redundant buttons on the UI
- Fully use json data when getting the information into the page via Jbuilder

### CODE REVIEW

**Comment by Joeper214** [comment]
This is quite a very long commit. because I was refactoring and setting a new UI goal for skyhopper. Is it okay if you can review this later? @root-beer

**Comment by Joeper214** [comment]
Please review and merge PR. @root-beer

**Comment by Joeper214** [comment]
Please review and Merge PR. @root-beer :bow: ..

**Comment by root-beer** [comment]
I'll review it now.

**Comment by root-beer** [comment]
It does not be reflected the information after deleting client or project.

**Comment by Joeper214** [comment]
:+1:  I think I haven't tested it on Firefox yet. I'll adjust it tomorrow. Thanks for reporting it.   :bow:

File: app/assets/javascripts/projects.js [hunk 1]
```diff
@@ -12,109 +12,51 @@
...
+      delete_entry: function()  {
+        var delete_project_url = this.picked.delete_project_url;
+        modal.Confirm(t('projects.project'), t('projects.msg.delete_project'), 'danger').done(function () {
+          $.ajax({
+            type: "POST",
+            url: delete_project_url,
+            dataType: "json",
+            data: {"_method":"delete"},
+          });
+          event.preventDefault();
```
Comment (root-beer) [inline app/assets/javascripts/projects.js]: An error occurred on Firefox.
`ReferenceError: event is not defined`

**Comment by Joeper214** [comment]
I think it's fixed. Please let me know if it's working. @root-beer .. ありがとう。:bow:

**Comment by root-beer** [comment]
It's working. Thanks! @Joeper214

**Comment by root-beer** [comment]
But it seems that user_admin page is not working...

**Comment by Joeper214** [comment]
:+1: Sorry. I'll fix it now.

**Comment by Joeper214** [comment]
I think I'm done fixing user_admin page. thanks @root-beer

**Comment by Joeper214** [comment]
Please fix iT18N. @root-beer おねがいします。:bow:

**Comment by root-beer** [comment]
LGTM! Thanks. @Joeper214 :bow:

**Comment by Joeper214** [comment]
:bow:

**Pull request merged**